src/Main.ls

```diff
@@ -64,7 +95,7 @@ package
             stage.scaleMode = StageScaleMode.NONE;
 
             // Set to false for development.
-            if(true)
+            if(false)
```

Comment:
Forgot to leave this out? :)

Maybe we could just have a file that we check for this. E.g. if a certain file exists treat it as dev build.